lib/railway_ipc/models/published_message.rb

```diff
@@ -0,0 +1,12 @@
+module RailwayIpc
+  class PublishedMessage < ActiveRecord::Base
+    self.table_name = 'railway_ipc_published_messages'
+    self.primary_key = "uuid"
```

Comment:
@aturkewi we need to do this here because Rails ended up requiring it when I found out from @StevenNunez that we need to remove the `id` column in the new table (matches the Hex package schema as well).